features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesView.kt

```diff
@@ -373,6 +381,13 @@ private fun MessagesViewTopBar(
                 )
             }
         },
+        actions = {
+            if (inRoomCallsEnabled) {
+                IconButton(onClick = onJoinCallClicked) {
+                    Icon(CommonDrawables.ic_compound_video_call, contentDescription = "Join call")
```

Comment:
i18n for the content description? This is maybe a temporary string, since the button is also used to create a call.